bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -46,18 +46,19 @@ <h2 class="c-sub-navigation-title">
 
   {% block channels %}{% endblock %}
 
-  {% call picto_card(
-    title=ftl('firefox-channel-see-something-that-isnt-working'),
+<ul class="mzp-t-picto-center">
```

Comment:
I'd suggest a `section` or maybe `aside` as the container here. A list would work, but it's a list of one in this case so I think `section` or `aside` is more logical.

Reply:
In the original picto card code, it was a `ul` and holding the 2 `li` elements with the `file a bug now` and `tips for filing a  bug`. Should I change it to `section`/`aside` and use `p` tags instead of the `li`?